packages/next/pages/_document.js

```diff
@@ -117,14 +117,15 @@ export class Head extends Component {
         }
         return child
       })
+      if (this.props.crossOrigin) console.warn('Warning: `Head` attribute `crossOrigin` is deprecated. https://err.sh/next.js/doc-crossorigin-deprecated')
```

Comment:
Below there is a `NODE_ENV` check, let's add it here too 💯

Reply:
It's in one :smile:

Reply:
Oh, doh 🤦‍♂️ the diff only showed part of the code lol